Detections/SecurityEvent/malware_in_recyclebin.yaml

```diff
@@ -21,7 +21,7 @@ relevantTechniques:
   - T1158
 query: |
 
-  let procList = dynamic(["cmd.exe","ftp.exe","schtasks.exe","powershell.exe","rundll32.exe","regsvr32.exe","msiexec.exe"]);
+  let procList = dynamic(["At.exe","Atbroker.exe","Bash.exe","Bitsadmin.exe","CertOC.exe","Cmstp.exe","Diskshadow.exe","Dllhost.exe","Dnscmd.exe","Explorer.exe","Extexport.exe","Forfiles.exe","Ftp.exe","Gpscript.exe","Hh.exe","Ie4uinit.exe","Ieexec.exe","Infdefaultinstall.exe","Installutil.exe","Mavinject.exe","Microsoft.Workflow.Compiler.exe","Mmc.exe","Msbuild.exe","Msconfig.exe","Msdt.exe","Msiexec.exe","Execute","Netsh.exe","Odbcconf.exe","OfflineScannerShell.exe","Pcalua.exe","Pcwrun.exe","Pnputil.exe","Presentationhost.exe","Rasautou.exe","Regasm.exe","Register-cimprovider.exe","Regsvcs.exe","Regsvr32.exe","Rundll32.exe","Runonce.exe","Runscripthelper.exe","Schtasks.exe","Scriptrunner.exe","SettingSyncHost.exe","Stordiag.exe","SyncAppvPublishingServer.exe","Ttdinject.exe","Tttracer.exe","Verclsid.exe","Wab.exe","Wmic.exe","WorkFolders.exe","wuauclt.exe","Xwizard.exe"]);
```

Comment:
Great, would it make more sense to automate a list from https://github.com/LOLBAS-Project/LOLBAS and then we can simply include it here?

Reply:
That's a great idea. I am not sure how to do it though. Do you mean to create a csv with these binaries and upload them or to automatically fetch them from the website?

Reply:
> That's a great idea. I am not sure how to do it though. Do you mean to create a csv with these binaries and upload them or to automatically fetch them from the website?

For now, it feels like it might make sense to just point to a CSV that we upload in the https://github.com/Azure/Azure-Sentinel/tree/master/Sample%20Data folder and use it in the query versus having a static list in the query.  Also update the description section with Reference to : https://github.com/LOLBAS-Project/LOLBAS so that it gives the user some idea about where we are sourcing these from.